baselines/dom.generated.d.ts

```diff
@@ -3448,7 +3448,11 @@ declare var Document: {
     new(): Document;
 }
 
-interface DocumentFragment extends Node, NodeSelector, ParentNode {
+interface DocumentFragment extends Node {
+    querySelector<K extends keyof ElementTagNameMap>(selectors: K): ElementTagNameMap[K] | null;
```

Comment:
I understand. but why remove `NodeSelector` from the extends clause, and then define the functions inline? 

it would have been sufficient to just remove `ParentNode` from the extends clause.